README.md

```diff
@@ -1 +1,23 @@
-# Hello-Microverse
\ No newline at end of file
+# Hello-Microverse
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+    - [Key Features](#key-features)
+- [👥 Authors](#authors)
+
+# 📖 [Hello-Microverse] <a name="about-project"></a>
+
+Hello-Microverse is a project that display a welcome message.
+
+### Key Features <a name="key-features"></a>
+
+- **[Display a welcome message]**
+- **[Use different color for the message than the default]**
+- **[Ignore unnecessary files]**
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Binyam Yohannes Olango**
+
+- GitHub: [@githubhandle](https://github.com/binyamolango)
```

Comment:
- [ ] Good job using the **Microverse** template for your `README`. However, I noticed that some sections are missing from your content table. Please ensure that you include all the main sections in your `README` as it is a requirement of the project to follow the [Microverse template](https://github.com/microverseinc/readme-template/blob/master/README.md). The only section that is optional to include is the `FAQ` section. Also, if your project doesn't require deployment, you can remove the `Live Demo` section. 🎯
**_If you don't have a logo you can remove it._**

  Please make sure your table of contents includes the following sections:

```
# 📗 Table of Contents

- [📖 About the Project](#about-project)
  - [🛠 Built With](#built-with)
    - [Tech Stack](#tech-stack)
    - [Key Features](#key-features)
  - [🚀 Live Demo](#live-demo)
- [💻 Getting Started](#getting-started)
  - [Setup](#setup)
  - [Prerequisites](#prerequisites)
  - [Install](#install)
  - [Usage](#usage)
  - [Run tests](#run-tests)
  - [Deployment](#deployment)
- [👥 Authors](#authors)
- [🔭 Future Features](#future-features)
- [🤝 Contributing](#contributing)
- [⭐️ Show your support](#support)
- [🙏 Acknowledgements](#acknowledgements)
- [❓ FAQ (OPTIONAL)](#faq)
- [📝 License](#license)
```

Reply:
@taldr27  I resolved it. Thanks for the comment.